stratum/hal/config/chassis_config_migrator.cc

```diff
@@ -31,6 +32,10 @@ ls -1 stratum/hal/config/*/chassis_config.pb.txt | \
     -chassis_config_file
 )USAGE";
 
+// Suppress gcc "no previous declaration" warnings.
```

Comment:
Not sure if we need this comment. Can get rid of it

Reply:
The comment is there to explain why there are prototypes for local functions.
It provides the non-obvious answer to the question, "why?"